megalinter/descriptors/lua.megalinter-descriptor.yml

```
@@ -54,3 +57,37 @@ linters:
       vscode:
         - name: vscode-luacheck
           url: https://marketplace.visualstudio.com/items?itemName=dwenegar.vscode-luacheck
+
+  # SELENE
+  - linter_name: selene
+    linter_url: https://kampfkarren.github.io/selene/
+    linter_repo: https://github.com/Kampfkarren/selene
+    linter_rules_url: https://kampfkarren.github.io/selene/lints/index.html
+    linter_rules_configuration_url: https://kampfkarren.github.io/selene/usage/configuration.html
+    linter_rules_inline_disable_url: https://kampfkarren.github.io/selene/usage/filtering.html
+    linter_megalinter_ref_url: "no"
+    config_file_name: selene.toml
+    cli_config_arg_name: "--config"
+    examples:
+      - "selene myfile.lua"
+      - "selene --config chktexrc.toml myfile.lua"
+    install:
+      apk:
+        - g++
+      dockerfile:
+        - |
+          RUN cargo install --branch main --git https://github.com/Kampfkarren/selene selene
```

We try to use official releases, so maybe try the following:

```suggestion
          RUN cargo install selene@0.27.1
```

---

Im still trying to find the correct syntax to keep the version number updated

---

For what I saw with `cargo` in the rust descriptor, it's just being omitted (just using the crate name):

https://github.com/oxsecurity/megalinter/blob/8ec5e3ef2d860b1bdf705e60f0e38a36d75c3e55/megalinter/descriptors/rust.megalinter-descriptor.yml#L23-L25

---

We weren't pinning packages up until a couple weeks ago, from my push to make keeping them updatable manageable. So you'll find lots of places where they aren't implemented yet

---

Do I leave it with `selene@0.27.1` or just `selene` for the time being.

---

We can try with this: https://github.com/oxsecurity/megalinter/pull/3978#discussion_r1750895746

---

@echoix LGTM 👍